README.md

```diff
@@ -12,32 +15,59 @@ Thoth is deployed to the following [environments](docs/environments.md).
 
 ## Separation of Application Components and Application Configuration
 
-Each component of the Thoth Application is separated into an ArgoCD Application, there is a core Application
-that contains global configs and secrets.
+Each component of the Thoth Application is separated into an ArgoCD Application.
+The core Application that contains global configs and secrets along with the setup of the namespace based requirements.
 
 ## Policy Based Control of resources
 
-All resources created by kustomize should pass OPA conftest, you could check this by
-using: `kustomize build --enable_alpha_plugins overlays/test | conftest test --policy ../policy -`.
+All resources created by kustomize should pass OPA conftest, you could check this by using: `kustomize build --enable_alpha_plugins overlays/test | conftest test --policy ../policy -`.
 
 ## Use Cases
 
-### testing a pull request
+### Test a pull request
 
-### pushing a release to production
+### Push Release to Production
 
 ### Developing a new component
 
+Developing a new component under thoth-station:
+
+- Please make use of the [template-project](https://github.com/thoth-station/template-project) it has basic requirements. Github Allow users to create new repository with template projects.
```

Comment:
add `,`  -> `,  it has basic requirements.` 

use `allows` -> `Github allows users to create new repository with template projects.`